src/components/directory-content.tsx

```diff
@@ -493,7 +496,7 @@ const DirectoryContent = () => {
         }
 
         // If empty dir then render an appropriate content
-        if (rows.length === 0) {
+        if (!rows || rows.length === 0) {
```

Comment:
why? `rows` can't be null or undefined by his type.  
~And if the type is not correct: `if (!(rows?.length > 0)) {`.~

Reply:
Changed